### ISSUE TITLE
jabref literature does not work anymore

### ISSUE THREAD

**Issue by mocherry** [JabRef/jabref#2253]
google scholar and medline searches reprot that either nothing is found or that there are errors performing the search????
Windows 7, latest jabref-version
see error log below.
Thanks

 

JabRef version  on 

Steps to reproduce:

1. ...
2. ...
3. ...

```
Put the excerpt of the log file here
13:42:12.129 [JabRef CachedThreadPool] WARN  net.sf.jabref.gui.importer.fetcher.GoogleScholarFetcher - Error fetching from Google Scholar
java.io.IOException: Server returned HTTP response code: 503 for URL: https://ipv4.google.com/sorry/index?continue=https://scholar.google.com/scholar%3Fhl%3Den%26oe%3DASCII%26num%3D20%26as_sdt%3D2006&hl=en&q=EgSE5i7XGKOQh8EFIhkA8aeDSwUT7RYs_1im9xcSXtUiV3gq7J4QMgFj
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source) ~[?:1.8.0_111]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source) ~[?:1.8.0_111]
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(Unknown Source) ~[?:1.8.0_111]
	at net.sf.jabref.logic.net.URLDownload.downloadToString(URLDownload.java:123) ~[JabRef-3.6.jar:?]
	at net.sf.jabref.gui.importer.fetcher.GoogleScholarFetcher.runConfig(GoogleScholarFetcher.java:166) ~[JabRef-3.6.jar:?]
	at net.sf.jabref.gui.importer.fetcher.GoogleScholarFetcher.processQueryGetPreview(GoogleScholarFetcher.java:82) ~[JabRef-3.6.jar:?]
	at net.sf.jabref.gui.importer.fetcher.GeneralFetcher.lambda$actionPerformed$4(GeneralFetcher.java:191) ~[JabRef-3.6.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_111]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_111]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_111]```

**Comment by stefan-kolb** [comment]
Thank you for your report :+1:
This should be fixed in current `master`. Please try the latest build from http://builds.jabref.org/master.
For the Google Scholar fetcher, please see https://github.com/JabRef/jabref/issues/2173.
You ran into an request limitation.

**Comment by matthiasgeiger** [comment]
Refs also https://github.com/JabRef/jabref/issues/2228

**Comment by hmontenegro** [comment]
Version JabRef-3.7-dev--snapshot--2016-11-08--master--fffad83.jar fixes Medline searches.

**Comment by koppor** [comment]
We released JabRef 3.7 and this should be fixed there. Please reopen, if not.
